src/Microsoft.AspNet.Session/CookieSecureOption.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNet.Session
+{
+    /// <summary>
+    /// Determines how the identity cookie's security property is set.
```

Comment:
BTW a lot of this new code mentions "identity" yet this is all about session. Is that deliberate?